specification/resourcegraph/resource-manager/Microsoft.ResourceGraph/stable/2019-04-01/resourcegraph.json

```diff
@@ -255,7 +267,7 @@
         },
         "data": {
           "description": "Query output in tabular format.",
-          "$ref": "#/definitions/Table"
+          "type": "object"
```

Comment:
why is the data type changing?

Reply:
Before we were returning response in Table format, but now it is either Table or IList<Dictionary>. Please let us know if you know better way of doing it.

Reply:
This looks like a breaking change

Reply:
yes. Changing response format is a breaking change. Please introduce new api version for these changes.
You could consider discriminator pattern that will allow ability to specify different properties based on "kind" property (top level property on arm resource).
I shall forward the details.. you may take a look


---
In reply to: [289210711](https://github.com/Azure/azure-rest-api-specs/pull/6148#discussion_r289210711) [](ancestors = 289210711)

Reply:
Added description with more details and explanation, why it won't break but require sdk update.